### ISSUE TITLE
issue with a2s-custom-types on fresh install

### ISSUE THREAD

**Issue by dmastag** [arnehilmann/markdeck#44]
After scaffolding my first project I am getting the following

```
~/markdeck
kursus-tuntas-ionic_markdeck_1 is up-to-date
kursus-tuntas-ionic_friends_1 is up-to-date
kursus-tuntas-ionic_web_1 is up-to-date
kursus-tuntas-ionic_pdf_1 is up-to-date
kursus-tuntas-ionic_standalone_1 is up-to-date
Creating kursus-tuntas-ionic_a2sketch_1 ... error

ERROR: for kursus-tuntas-ionic_a2sketch_1  Cannot create container for service a2sketch: invalid mount config for type "bind": bind source path does not exist: /Users/julianalimin/projects/kursus-tuntas-ionic/a2s-custom-types

ERROR: for a2sketch  Cannot create container for service a2sketch: invalid mount config for type "bind": bind source path does not exist: /Users/julianalimin/projects/kursus-tuntas-ionic/a2s-custom-types
ERROR: Encountered errors while bringing up the project.
```

So I tried to create the folder `a2s-custom-types`

And when I run `~/markdeck` to compile again I am getting the following in http://localhost:8080

```
call render_a2s
curl: (6) Could not resolve host: a2sketch
Error running filter /markdeck/lib/render-asciiart-filter.lua:
Error running curl (error code 6): 
stack traceback:
	[C]: in field 'pipe'
	/markdeck/lib/render-asciiart-filter.lua:170: in function 'Render'
	/markdeck/lib/render-asciiart-filter.lua:185: in function 'RenderCodeBlock'
```

Any help would be appreciated

**Comment by dmastag** [comment]
As you can see I tried to put markdeck in my home folder, so I tried from scratch and instead put markdeck in the project folder (also into the git repo).

I am still getting the complaint regarding having no `a2s-custom-types` folder. But this time after I added the folder, I could successfully access the presentation at http://localhost:8080

Still would like to know if it is normal to have to add that folder manually?

**Comment by arnehilmann** [comment]
Hi! you are right: adding that folder manually is not meant to be the normal way of starting markdeck, but a workaround:
stopping markdeck, adding that folder, starting markdeck again...
Do you have the `markdeck` executable in the same folder as your presentation? If not, please copy it there and run `./markdeck` in the same folder as your `slides.md` (and that dreaded `a2s-custom-types` folder).

**Comment by dmastag** [comment]
Hi @arnehilmann ,
Yes, I put it into the same folder as my presentation.

It works that way

**Comment by arnehilmann** [comment]
should be fixed with v0.53.1
